### PR TITLE
feat(Feedback): Add support for tooltips

### LINKED ISSUE #5188: Add FormControl.Feedback tooltip variant
## Is your feature request related to a problem? Please describe

Noticed the FormControl.Feedback was missing the tooltip variant. See official bootstrap docs:
https://getbootstrap.com/docs/4.5/components/forms/#tooltips

## Describe the solution you'd like

I could create a PR for this. Perhaps adding a `tooltip` boolean prop to Feedback that would set the tooltip class if true and original feedback class if false/undefined?

### PR DESCRIPTION
**Pull request by kyletsang**
Resolves #5188

### CODE REVIEW

File: src/FormCheck.js [hunk 1]
```diff
@@ -79,6 +79,9 @@ const propTypes = {
   /** Manually style the input as invalid */
   isInvalid: PropTypes.bool.isRequired,
 
+  /** Display feedback as a tooltip. */
+  tooltip: PropTypes.bool,
```
Comment (taion) [inline src/FormCheck.js:83]: **suggestion:** maybe this should be something like `feedbackTooltip`? it'd be more explicit given context

Comment (kyletsang) [reply src/FormCheck.js]: Sounds good. I changed this in both FormCheck and FormFile

**Comment by jquense** [comment]
awesome thank you!

**Comment by shaungc-su** [comment]
Nice work! Wondering would this be included in the next release like `1.1.0`?

**Comment by kyletsang** [comment]
> 
> 
> Nice work, looking forward to using the tooltip variant! Wondering would this be included in the next release like `1.1.0`? Looks like it's not yet included in`v1.1.0-rc.0`.

@taion?

**Comment by taion** [comment]
@kyletsang looks like @jquense is already setting up for 1.1.0 in https://github.com/react-bootstrap/react-bootstrap/pull/5251, so maybe we just wait until then?

**Pull request merged**